### ISSUE TITLE
Escape region names

### ISSUE THREAD

**Issue by Xairoo** [digitalcreations/MaxTo#697]
You should escape region names.

I have 3 regions (just shorted for this example)

- Full
- 1/1
- 1460, 2380

The last name will run you into problem. Just escape the names 

```
{
    "command": "regions:cycle",
    "parameters": {
        "presets": "Full,1/1,1460, 2380",
        "monitor": "97F8P91C06UL"
    }
}
```

Should be something like:
`"presets": "'Full', '1/1', '1460, 2380'",`

**Comment by vegardlarsen** [comment]
This should really have been an array in the JSON, not a string. This will be fixed. :)

**Comment by vegardlarsen** [comment]
I have fixed this for the next 2.2.0-release (maybe another beta, maybe final).

Just while writing this comment, I discovered that this change breaks previously correct configurations (e.g. normal preset names without commas in them will be broken), so I am reopening and will fix this as well.

**Comment by vegardlarsen** [comment]
Now it is solved. :)
